config/verification-code.php

```diff
@@ -48,6 +48,19 @@
     */
     'expire_hours' => 1,
 
+    /*
+    |--------------------------------------------------------------------------
+    | Custom notification
+    |--------------------------------------------------------------------------
+    |
+    | This class contains the notification sent to users upon creation
+    | of the verification code.
+    |
+    | It should implement Illuminate\Contracts\Queue\ShouldQueue and NextApps\VerificationCode\Notifications\VerificationCodeInterface.
+    |
```

Comment:
Maybe we can write this out in the following way
```suggestion
    | It should implement:
    | - Illuminate\Contracts\Queue\ShouldQueue
    | - NextApps\VerificationCode\Notifications\VerificationCodeInterface
```